src/applications/disability-benefits/wizard/pages/unable-to-file-bdd.jsx

```diff
@@ -15,34 +16,31 @@ function alertContent(getPageStateFromPageName) {
   });
   const dateToday = moment();
   const differenceBetweenDatesInDays =
-    dateDischarge.diff(dateToday, 'days') + 1;
+    dateDischarge.diff(dateToday, 'days') - 179;
+  const dateEligible = dateToday
+    .add(differenceBetweenDatesInDays, 'days')
+    .format('MMMM D, YYYY');
```

Comment:
Same date question here, wondering if it follows convention here - https://design.va.gov/content-style-guide/dates-and-numbers

Reply:
Full month names are better for a11y too! 😸